applicationinsights.dev.json

```diff
@@ -16,16 +16,49 @@
   "sampling": {
     "percentage": 100,
     "overrides": [
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.system",
+            "value": "postgresql",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 100
+      },
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "server.address",
+            "value": "(sign-in|prison-api).*",
```

Comment:
only requests to prison api are for `GET /api/users/me` and auth are `POST /auth/oauth/token`